### DISCUSSION TITLE
Datatable search for arabic language not work!

### DISCUSSION THREAD

**Discussion by mohjacksi** [spatie/laravel-translatable#359] [Q&A]
Hello,
the data in the database is stored like this:
{"ar":"\u0628\u063a\u062f\u0627\u062f","en":"Baghdad","kr":"\u0628\u06d5\u063a\u062f\u0627"}
and the datatable server side search like this:
{
                "data": "name",
                "name": "name",
                "searchable": "true",
                "orderable": "true",
                "search": {
                    "value": "أربيل",
                    "regex": "false"
                }
            },

The problem is when looking for أربيل it's not found because of characters, this is the query from datatable
            "query": "select `cities`.* from `cities` where LOWER(`cities`.`name`) LIKE ? and `cities`.`deleted_at` is null order by `name` asc limit 25 offset 0",

**Comment by malozaibi** [comment]
I had the same problem, the solution was to change the data type of the schema from `text` to `json`.

Before
```
Schema::create('news_posts', function (Blueprint $table) {
            $table->id();
            $table->text('title');
            $table->text('body');
```

after
```
Schema::create('news_posts', function (Blueprint $table) {
            $table->id();
            $table->json('title');
            $table->json('body');
```

you can query with [see here](https://spatie.be/docs/laravel-translatable/v6/basic-usage/querying-translations)
